mackerel/api.go

```diff
@@ -56,12 +57,17 @@ func infoError(msg string) *InfoError {
 }
 
 // NewAPI creates a new instance of API.
-func NewAPI(rawurl string, apiKey string, verbose bool) (*API, error) {
+func NewAPI(rawurl string, apiKey string, verbose bool, disableKeepAlive bool) (*API, error) {
 	c, err := mkr.NewClientWithOptions(apiKey, rawurl, verbose)
 	if err != nil {
 		return nil, err
 	}
 	c.PrioritizedLogger = logger
+	t := http.DefaultTransport.(*http.Transport).Clone()
+	print(disableKeepAlive)
+	t.DisableKeepAlives = disableKeepAlive
+	c.HTTPClient.Transport = t
```

Comment:
Since `http.DefaultTransport` is made from `http.Transport`, we can cast it here.
https://pkg.go.dev/net/http#:~:text=DefaultTransport%20is%20the,lowercase%20versions%20thereof